modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/RaftException.java

```diff
@@ -15,38 +15,40 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.raft.client.exception;
+package org.apache.ignite.raft.jraft.rpc.impl;
 
 import org.apache.ignite.lang.IgniteInternalException;
-import org.apache.ignite.raft.client.RaftErrorCode;
+import org.apache.ignite.raft.jraft.error.RaftError;
 
 /**
  * A raft exception containing code and description.
  */
 public class RaftException extends IgniteInternalException {
```

Comment:
Did you consider using IgniteInternalCheckedException exception instead?

Reply:
Not sure, maybe we should use the usual org.apache.ignite.raft.jraft.error.RaftException instead of our custom.

Reply:
My only point is to use checked exception here.